OreshkoJA/lab1

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+using namespace std;
+int main()
+{
+	setlocale(0, "");
+	float first, second;
+	cout << "Введите первое число:";
+	cin >> first;
+	char sign;
+	cout << "Введите знак:";
+	cin >> sign;
+	cout << "Введите второе число:";
+	cin >> second;
+	float result;
+	if (second == 0) {
+		if (sign == '/') {
+			cout << "Делить на ноль нельзя" << endl;
+			system("pause");
```

Comment:
Avoid code copying
You wrote "system("pause");" 3 times, when there should be only one: at the end of application lifetime right before closing